scRNA-seq/post-processing_scRNA-seq.Rmd

```diff
@@ -0,0 +1,98 @@
+---
+title: "scRNA-seq_post_processing"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+**CCDL 2019**
+
+In this notebook, we'll perform quality control analyses and normalization of 
+scRNA-seq count data. 
+
+As opposed to bulk rna-seq, there are there are a few main things to look out for
+in single cell RNA-seq:
+
+**Single-cell rna-seq has...**  
```

Comment:
```suggestion
**Single-cell RNA-seq has...**  
```

Reply:
Is it `single-cell` or `single cell`? You use both. I also think this should be **Single cell RNA-seq:** (this is not me weighing in on the style of `single cell`)

Reply:
In papers I see it both ways, but I think I see it with the `-` more than without, so I've gone back and added `-` everywhere.